website/content/docs/job-specification/parameterized.mdx

```diff
@@ -152,9 +152,59 @@ job "email-blast" {
 }
 ```
 
+### Interactions with [`periodic`][periodic] 
```

Comment:
```suggestion
### Use periodic with parameterized
```

Changed the header because I think users would search for "use" more than "interactions with".
I moved the `[periodic]` link to L157.

I also added a direct link to this section in the periodic.mdx file L122, so if you don't accept my changes, please update that entry as well.